cage.c

```diff
@@ -65,6 +65,22 @@ spawn_primary_client(char *argv[], pid_t *pid_out)
 	return true;
 }
 
+static bool drop_permissions(void)
```

Comment:
Can you please align the formatting with the rest of Cage and insert a newline after `static bool`?